.github/workflows/ci.yaml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on:
+  push:
+    pull_request:
+      types: [opened, reopened]
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+jobs:
+  Security:
+    name: Security Pipeline
+    uses: uc-cdis/.github/.github/workflows/securitypipeline.yaml@master
+    with:
+       python-poetry: 'false'
+    secrets: inherit
+
+  UnitTest:
+    name: Python Unit Test with Postgres
+    uses: uc-cdis/.github/.github/workflows/python_unit_test.yaml@master
```

Comment:
the default script runs `poetry run pytest -vv` but in indexd we have coveralls set up and we want to report coverage updates, so i think we actually need a custom script that does this: https://github.com/uc-cdis/indexd/blob/2023.09/.travis.yml#L20-L23

Reply:
I'd actually be okay leaving this as-is, because if we want to add consistent coverage in the future, we could just modify the template. but we'd love coverage reporting in the meantime. idk, I don't have a strong opinion either way

Reply:
It seems we've lost coverage reporting already, checking the last few closed PRs for indexd. 

Checking this PR https://github.com/uc-cdis/indexd/pull/361 and this one https://github.com/uc-cdis/indexd/pull/360 there's no coveralls run. It seems the last one was Atharva's PR here: https://github.com/uc-cdis/indexd/pull/359 

I could be missing something, but it seems we've already lost coverage reporting so my vote is to add it to the template in a separate PR. Sound good @paulineribeyre ??

Reply:
I don't know why coveralls is not reporting as a check on those PRs, but you can see in coveralls that it did get the report, for example for #361:
<img width="1129" alt="Screenshot 2023-09-28 at 4 00 33 PM" src="https://github.com/uc-cdis/indexd/assets/4224001/36eaa0c3-fce0-4253-9e7e-7d266ef5373a">

But anyway i'm referring to the coverage badge we have on this repo's readme, more than to the PR checks. It would be nice to include the coveralls logic in the template, but since it's not there yet, i think removing it would be a regression and the coverage % on the repo readme would be inaccurate. Tbh i think it's probably not a heavy lift to add it, but if it is for some reason, we can file a ticket instead

Reply:
Ah, that makes sense. Given that https://github.com/uc-cdis/sheepdog#sheepdog has broken tags now I'll see what I can do about getting this working here and moving it to the template. 

Reading the docs here https://github.com/marketplace/actions/coveralls-github-action it looks fairly straightforward but I might have to DM you with questions if I run into something I can't find in the coveralls config or guides.